<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boardWidget</class>
 <widget class="QWidget" name="boardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#boardWidget {
    background-color: qlineargradient(spread:pad, x1:0.527222, y1:0.961, x2:0.536946, y2:0.198864, stop:0.103448 rgba(165, 165, 165, 255), stop:1 rgba(249, 249, 249, 255));
    border-radius: 16px;
    border: 2px solid #CCCCCC;
}

QLabel#boardNameLabel {
	font-size: 20pt;
	font-weight: bold;
}

QLabel#statusLabel {
    background-color: none;
    border: none;
	font-size: 12pt;
	color: #888888;
}

QPushButton#flashButton {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #73D95D, stop: 1 #18A154);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="40,40,10,10" columnstretch="40,20,10">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="iconLabel">
     <property name="text">
      <string>ICON</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="boardNameLabel">
     <property name="text">
      <string>PX4FMU v1.x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Fetching Firmware options..</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="firmwareComboBox"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="flashButton">
     <property name="text">
      <string>Upgrade</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="bootloaderLabel">
     <property name="text">
      <string>Bootloader v 3.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
